dart/tool/common/generate_common.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library generate_vm_service_lib_common;
+
+import 'package:markdown/markdown.dart';
+
+import 'src_gen_common.dart';
+
+var _NINE = '9'.codeUnitAt(0);
+var _ZERO = '0'.codeUnitAt(0);
+
+/// [ApiParseUtil] contains top level parsing utilities
```

Comment:
It might be helpful to add a sample of the text you're trying to parse here. I assume it's:

```
# Dart VM Service Protocol 2.0
```

Also, this seems like a good place to use a regex :) The right regex would make very short work of this.